O21.Game/U95/Parser.fs

```diff
@@ -7,31 +7,34 @@ type MapOfLevel =
     | Bomb // random: octopus or bomb
     | Bonus // random bonus
     | Empty
-    
+
 type Parser(directory) =
-     member private this.readDatFile(level:int) (part:int): string[] = 
-        let lines = File.ReadAllLines(Path.Combine(directory, $"U95_{level}_{part}.DAT")) 
+    member private this.readDatFile (level: int) (part: int) : string[] =
+        let lines = File.ReadAllLines(Path.Combine(directory, $"U95_{level}_{part}.DAT"))
         lines
-       
-     member private this.parseLine (line:string): MapOfLevel[] =
-         let levelMap = Array.init line.Length (fun i ->
-            let brick = 
-                match line[i] with
-                    | t when t>='1' && t <= '9' -> Brick(int line[i] - int '0')
+
+    member private this.parseLine(line: string) : MapOfLevel[] =
+        let levelMap =
+            Array.init line.Length (fun i ->
+                let brick =
+                    match line[i] with
+                    | t when t >= '1' && t <= '9' -> Brick(int line[i] - int '0')
                     | 'b' -> Bomb
                     | 'a' -> Bonus
                     | _ -> Empty
-            brick
-         )
-         levelMap
-                
-     member private this.parseLevel (data:string[]): MapOfLevel[][] = 
-         let level = Array.init data.Length (fun i ->
-            let bricks = this.parseLine data[i]
-            bricks
-         )
-         level
-
-     member this.LoadLevel (level:int) (part:int): MapOfLevel[][] =
+
+                brick)
```

Comment:
Note for me: let's go and create an issue for Fantomas to respect this closing paren as-is.

Reply:
At least for lambdas we can use [fsharp_multi_line_lambda_closing_newline](https://fsprojects.github.io/fantomas/docs/end-users/Configuration.html#fsharp_multi_line_lambda_closing_newline) as workaround.

Reply:
It doesn't look good: for whatever reason, [it reformats the whole expression and makes it ugly](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKEMYDaIC5hwDdEsBZAQwAcwBeAHQCc6A7TNjAQQYfIE8A6AJbNB%2BLMMT8AMomYBzXAAswACgBmAV1aCwAWgB8jFuxPY8YAEYNB0ANa0jrU84gBbcrmjLxzRAG1BAF0wAHdRRUcXFwAfMHwQxVk4sH0aMAByAEZ0sAAyXOSAHjT0gE4cgzAAIWs7FWExCQDg3TAGjIAGdIBKSKjTWPSLCv1q2FcLPv72QfIRseYNAGcp6cxYgH09UYBRV0pcXkdV-qsbW27IABpwKCW8XGE5JdQwP1uIUHZIABJDymQaEgDWuHwwv0I5CwGkBbzBmA6N1MkAAkswkMxcABlQQAL2QSJMABZ4WBAmCAL6Ez6k37-WHAzGg74-SHQ2HvUyZalsSAUAAeUgkMnkSmZJkyACYOqTyZgqWCvry-rwAa9ILJ4AB5NRC3xYw44cWYCFQmGvTkmSU8k07dE6vUE0kQSBYNSQWWUnlKk0qtVAiywWBYY3g1lmjnOsAAZht4LR9wYuAAYsIoQA5RAhHxO0y4Bgwz3y720v0MwPB0Mu8Psi1Rolxl1YyjkaCIKqINSwBiIAAK5B4rjwiAYVYw%2BcL7DlGAVmB9Yfp6orIbjptraEt7AArI2oM3W%2B3O93EFJYCER9ByPc0YRYJfHrBmGOIBPEEWZyWWYuA0GV6W2eaG5RgAbLukD7m2HZdj2ACqlAAgwl7Xswt73oIj7PmAahQvc74QLOGDztW35QMuY5roBcKmAA7GBEGHtBiAAMJYFeSxMY%2BSwTtAuDdph2FYLhU5eoqpYkZAZGrjWlGbmwAAcdEtpBR49qQiATCO-E4W%2BwnFqJX6quWv7kdJkamKUikHlBx4cVgGG7gJQlsNO%2BGfsq4mSf%2BEZ1lyiJRuBSmIBwai4COHGuO4mGvnhYAETSBn%2BqRxlSQBZkSty-n0dZPZYupNjBvZUaOTpzkiXOYmGUuyVeeuVEStamWBcFoUMLlrj5XZT67tFukfvp7mVT%2BlYpd5QFcrGjUHlwsBaPAAAiJCCO1LVRQWJWYC5sVub64kgiNtWyZgmQNv5AoomoAAqiTMFiijdrgADqgjwGKu4yr1rn9Ttg1QHtNUyVGmQ7qd5D8udV2yDsgmILd91PS9iiYcB72lXp5UJQyf0sqlPkSqBINg8waiCPyWoIR43Y7PylA9ksSzoV1UZySjG1lYRFWJYyuAmTjY0SrRBMAEqIHADDwPDr31izGCbXFYBEVAZbqljyq83V7CZApQsi928DphoGkMDqKKha4Ly7pkMVywrdI-ZArgaFgjw5sm3buLgLUXZV%2B0A1yFn%2BcLovwKQjvOxIrsMO7K1gV4A6ti1GxhAjHofVtX0LnbKu%2Bmrh0YNKYEClwPC8FqDBClxEuI7uzNW9tGec1nYY51GkoZcihfcHwpfl7g%2BuG8bpvm4Dtfp8RdsO07ggu27Hie97-1pewkoNciRdd2XghcSHk-T5Hs%2BaTHihxzxI6J89Yoj%2BjA0N0yPuL2wkoTe3oMAGoRlUwjwE8leYTXqfWxzTGt8F64yXidZ%2B-JkxaB4gzD%2B6Jv7nyrlLS%2B7MMbK2AdjUa6sH7AwgXNWAuAADieAqY00QHTBmP9q7SwgLLOuY9OaeUwQdFu%2BNkSZmzBIDsuBzyyC9gCBaxMRAPmYBwdEalDZDzzGtFB8Vr5GWGiAvmS8Bar3EHIZgUDmAwMfDiDRHgNA9gurANEGJcAeAZlpaGsj5aAKqoo5hvsrRazUS1ZgFjiCnnkBIiwI5BHCFEAzKRJhio2JtkrIaf5HH30wJKf27dQ6CEdFIcgEx4DkBYrAem8gOE5lWpOVGfUr7fUYdVaJoC2DRj8siUhI4lqyHMVgAA0ogRAlBTENLRDUcg2ikGmFCf-ehisPJlNVlg3OEBoxtxMJAKorFmC2EdEsaas1MxcUQMHRJOZSG03ppxfJ60ZZszkSUhRUSxksNMNGFeMyqgDmyogOam9oDWHau40K8AYIiEfAtaArEeAiKsU5VmaNUHyPVBPMOvhul2DwAaXgRo74VMwNGJ%2BMzt5QvbDwWF2JDS5hmS81JAJ4Ap0KZ9Yp9cgHcyRcoyp4CZktLaQKPuvijZqDmT0xZEhgkaw6DQskxzbFoMiTzcZUZoy4JmbkrhKlEAYqnhIcKlANDmJETs8hezGbSIKSCopYLTn2POdnMVVy2EzNqa8hpUIobtSWH0kJ2kra3ECISYESxkz01eKEikQA).

I don't think we should use that, but should find or create a feature request instead.